pkg/3scale/amp/operator/system_reconciler.go

```diff
@@ -206,25 +207,13 @@ func (r *SystemReconciler) Reconcile() (reconcile.Result, error) {
 			r.systemZyncEnvVarMutator,
 			r.systemDatabaseTLSEnvVarMutator,
 			r.systemRedisTLSEnvVarMutator,
+			reconcilers.DeploymentVolumesMutator,
```

Comment:
If we are doing it as part of mutators, can we have a situation where I would add a custom volume and it will get reverted back?

Reply:
same for 2 lines below.